src/pages/AfterLogin/SearchInput.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+const SearchInput = () => {
```

Comment:
couldn't you have used the existing  re-usable input instead of creating a brand new one ?